modules/nf-core/midas/run/main.nf

```diff
@@ -0,0 +1,71 @@
+VERSION = '1.3.2' // WARN: Version information not provided by tool on CLI. Please update this string when bumping container versions.
```

Comment:
The `VERSION` variable should be inside the script part of the module, see: https://nf-co.re/docs/contributing/modules#general paragraph 4 :)